internal/csi-common/utils_test.go

```diff
@@ -177,3 +177,219 @@ func TestIsBlockMultiNode(t *testing.T) {
 		assert.Equal(t, isMultiNode, test.isMultiNode, test.name)
 	}
 }
+
+func TestIsMultiWriterBlockRWOFile(t *testing.T) {
```

Comment:
The size of this unit-test should have suggested that `IsMultiWriterBlockRWOFile()` is not a very practical function to use. Make the code simpler by splitting `IsMultiWriterBlockRWOFile()` in separate functions please.

Reply:
Done. segregated into different functions.